ci/launch_cluster.sh

```diff
@@ -33,24 +46,36 @@ wget 'https://downloads.dcos.io/dcos-test-utils/bin/linux/dcos-launch' && chmod
 envsubst <<EOF > config.yaml
 ---
 launch_config_version: 1
-template_url: $TEMPLATE
 deployment_name: $DEPLOYMENT_NAME
-provider: aws
+platform: aws
+instance_type: m4.xlarge
+os_name: cent-os-7-dcos-prereqs
 aws_region: us-west-2
+aws_key_name: default
+provider: onprem
+installer_url: $INSTALLER
```

Comment:
How about using `$INSTALLER_URL` instead of `$INSTALLER`?

Reply:
Yup. Good idea 👍